test/evergreen.yml

```diff
@@ -740,7 +748,7 @@ tasks:
       - func: "get project"
       - func: "compile wiredtiger"
         vars:
-          posix_configure_flags: --enable-silent-rules --enable-strict --disable-shared
+          posix_configure_flags: -DENABLE_STRICT=1 -DENABLE_SHARED=0 -DENABLE_STATIC=1
```

Comment:
I'm wondering how `--enable-silent-rules` gets mapped in the CMake setting.

Reply:
Good question, CMake configured builds by default are silent (for both `ninja` and `make`), hence I didn't feel a need to specify an equivalent to `--enable-silent-rules` for our CMake. To get a verbose build, the option is deferred the actual invocation of the build system (`ninja -v` or `make VERBOSE=1`)